src/components/mods/RandomizeWildPokemonMod.ts

```diff
@@ -0,0 +1,50 @@
+import { Component } from "eightbittr/lib/Component";
+import { ICallbackRegister, IMod } from "modattachr/lib/IModAttachr";
+
+import { FullScreenPokemon } from "../../FullScreenPokemon";
+import { IWildPokemonSchema } from "../Maps";
+
+/**
+ * Mod that randomizes Pokemon encounters based on the original wild Pokemon's
+ * type.
```

Comment:
Put `type.` on the same line. The comment won't be too long that way (it's less than the class declaration width)